Add contact form.

emiliazm opened the pull request:
Hello!

In this pull request the changes are:
-Add Contact section.
-Make it flex.
-Add new elements to section such as img, h1, form, and buttons and also flex some of them.
-Add a footer section and frex it.
-Add new elements to it such as a p element and some img.

Looking forward to your comments.

Emilia

emiliazm committed:
Fix hover and active buttons

emiliazm committed:
Add contact section and add first input in form

abelagza committed:
Emilia form fixes

emiliazm committed:
Add new inputs and add style to them

abelagza committed:
Add title&subtitle to contact form section

emiliazm committed:
Add footer section and style it

emiliazm committed:
Fix buttons in all sections

emiliazm committed:
Fix cvBut button

emiliazm committed:
Set 30 chars to name on forma in contact section

emiliazm committed:
Fix stylelint erros

ShadyShawkat commented on style.css:
```diff
@@ -227,3 +268,168 @@ h3 {
+}
+
+label {
+  position: relative;
+  display: block;
+  box-sizing: border-box;
+}
+
+label::after {
+  position: absolute;
+  content: ' ' attr(data-domain);
+  top: 4px;
+  left: 20px;
+  font-size: 10px;
+  font-family: 'Poppins', sans-serif;
+  color: #7d7c84;
+}
+
+.text-area {
+  font-family: 'Poppins', sans-serif;
+  color: #7d7c84;
+  width: 282px;
+  height: 112px;
+  font-size: 14px;
+  border: 1px solid #dfdfe0;
+}
+
+.text-area::-webkit-input-placeholder {
+  padding-top: 10px;
+  padding-left: 16px;
+}
+
+.form {
+  margin: 12px 14px 60px 18px;
+  width: 288px;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  gap: 8px;
+}
```
- [ ] Kindly try to refrain from using fixed widths on containers as much as possible. You can use `width: 100%` instead so that your design would match the Figma design. 

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/159683906-07457a40-d9fd-4bcd-81c3-27b5ec7aa001.png) | ![image](https://user-images.githubusercontent.com/62290545/159684062-5a0839dc-4192-4f04-86c4-6ad4ef653aed.png) |

emiliazm replied:
Hello @ShadyShawkat ! Thank you for your comments.
Using 100% doesn't work for my form. Should I use a media query?
Looking forward to your comments.
Emilia

ShadyShawkat replied:
Hey @emiliazm,
You have to use `width: 100%` on the form and on each input

ShadyShawkat commented on index.html:
```diff
@@ -147,10 +147,45 @@ <h3>Skills</h3>
+
+    <section class="contact">
+        <img class="pic1" src="./images/image_1.png" alt="logo">
+        <h1 class="get-st" >Get Started</h1>
```
- [ ] [OPTIONAL] Please change this `h1` tag to any other heading tag `t2`-`t6`. Your web page should contain only one `h1` tag for the main title of the page and then other heading tags depending on the hierarchy of the headings.

ShadyShawkat reviewed (changes requested):
## Hi @emiliazm and @abelagza, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

![Image](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Kindly edit your 'Authors' section in your `README.md` file to have your name instead of 'Author1`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

emiliazm committed:
Modify width in contact section and modify Readme file

emiliazm commented:
Hello @ShadyShawkat!
Thank you for your comments!
I made the following changes:
I changed the width of the contact form.
I changed the README file in Author.

Looking forward for your comments.
Emilia

ShadyShawkat reviewed (approved):
## Hi @emiliazm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Image](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
